tests/php/Functional/WC/Payment/Mollie_WC_Payment_Order_Test.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Mollie\WooCommerceTests\Functional\Payment;
+
+use Mollie\Api\Resources\Order;
+use Mollie\WooCommerceTests\TestCase;
+
+use Mollie_WC_Helper_Data;
+use Mollie_WC_Payment_Order;
+use Mollie_WC_Payment_OrderItemsRefunder;
+use WC_Order;
+
+use function Brain\Monkey\Functions\when;
+
+
+class Mollie_WC_Payment_Order_Test extends TestCase
```

Comment:
I would not use `buildTesteeMock` for every mock you want to create. 
That helper function was created to help creating the SUT/Testee class and changes we'll do on that method will be  done by taking in mind that we are dealing with SUT not other classes.

If you want to create other classes mock use the phpunit methods directly.